dev/io.openliberty.grpc.1.0/bnd.bnd

```diff
@@ -11,19 +11,20 @@
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion: 1.0
 
-Bundle-Name: WebSphere gRPC API
-Bundle-Description: WebSphere gRPC API, version ${bVersion}
-Bundle-SymbolicName: io.openliberty.grpc
+Bundle-Name: Open Liberty gRPC API
+Bundle-Description: Open Liberty gRPC API, version ${bVersion}
+Bundle-SymbolicName: io.openliberty.grpc.1.0
 
-Import-Package: io.openliberty.grpc.annotation
+Import-Package: \
+  io.grpc, \
+  io.openliberty.grpc.annotation
 
-Export-Package: io.openliberty.grpc.annotation
+Export-Package: \
+  io.openliberty.grpc.annotation
```

Comment:
This project seems slightly odd. It looks like it's not used at all as a bundle and just copies in the `io.openliberty.frpc.annotation` package so that it can be put in the `dev/api` directory as a .jar?

I think that's ok, just seemed slightly odd.

Reply:
The only purpose of this package is to provide the IBM API for `grpc-1.0`, which is just a single package (and class!)  It is a little odd, but from the examples I looked at this is the correct way to create the `dev/api` jar.